forms.rst

```diff
@@ -695,6 +695,14 @@ the choice is ultimately up to you.
     to modify it, use the :method:`Symfony\\Component\\Form\\FormFactoryInterface::createNamed` method.
     You can even suppress the name completely by setting it to an empty string.
 
+Using Data Transfer Objects
+---------------------------
+
+There are some problems when using entities as directly mapped data classes for forms.
```

Comment:
Maybe not the word problems, but notice or info 
Better write something like :

Alternatively, you can decouple your entities creation with DTO that are bound to form instead, see more
Etc